specification/components/schemas/RelatedPersonBundle.yaml

```diff
@@ -10,7 +10,7 @@ properties:
     default: searchset
   timestamp:
     type: string
-    description: Time the search was performed.
+    description: Time the search was performed, including the year, month, date, time and timezone, in the format YYYY-MM-DDThh:mm:ss+hh:mm.
     format: datetime
     example: '2019-12-25T12:00:00+00:00'
```

Comment:
As fan of not repeating ourselves when the specification is clear and explicit - is the additional text not already made clear and explicit in the `format` and `example` field?

Reply:
[This is a requirement from a previous ticket](https://nhsd-jira.digital.nhs.uk/browse/SPINEDEM-2537) - AC5. Approved by TW

Reply:
Happy to leave as is, but, as I have just found out, if we change it to `format: date-time`, this would mean that it is only ever `YYYY-MM-DDThh:mm:ss+hh:mm` format - see https://swagger.io/docs/specification/data-models/data-types/#string which leads to https://datatracker.ietf.org/doc/html/rfc3339#section-5.6

Reply:
Okay interesting - are you saying we're planning to change the format to `date-time` rather than `datetime`? Or is that already the case and we need to make sure this OAS file matches it now?